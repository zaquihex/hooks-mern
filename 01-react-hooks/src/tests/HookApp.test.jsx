import React from 'react';
import { shallow } from 'enzyme';
import { HooksApp } from '../HooksApp';
describe('Pruebas de <HookApp />', () => {

    test('debe mostrarse correctamente', () => {
        const wrapper = shallow(<HooksApp />);
        expect(wrapper).toMatchSnapshot();
    })
});