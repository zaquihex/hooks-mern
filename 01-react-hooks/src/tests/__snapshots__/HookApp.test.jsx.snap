// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas de <HookApp /> debe mostrarse correctamente 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
